src/components/containers/updatePassword.js

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable no-useless-escape */
+/* eslint-disable react/no-unused-state */
+/* eslint-disable no-lone-blocks */
+import React, { Component } from "react";
+import UpdatePasswordView from "../views/UpdatePasswordForm";
+import ResetPasswordAPI from "../../api/resetPasswordAPI";
+
+class UpdatePasswordPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      password: "",
+      confirmPassword: "",
+      passwordInputError: "",
+      confirmPasswordInputError: "",
+      passwordButton: true
+    };
+  }
+
+  onInputChange = (event) => {
```

Comment:
can you change the logic in this function and set the email to state if it is valid

Reply:
@justmesam  I have applied the review, thank you.